skill.ts

```diff
@@ -24,6 +24,8 @@ import {
 } from "@atomist/skill";
 
 export const Skill = skill({
+	displayName: "Docker Build",
+	description: "Build Docker images and push them to a Docker registry",
```

Comment:
If you move those here please also move author, license etc from package.json.

Reply:
I moved this to hopefully fix [this](https://go.atomist.com/catalog/skills/atomist/docker-build-skill):

<img width="787" alt="description-displayName" src="https://user-images.githubusercontent.com/57881/97644179-a9300a80-1a17-11eb-9de5-23960f191351.png">

Without these being set here, it seems the web app is using the package.json description as the display name. I don't know if that is coming from the registration or what, but it doesn't seem right. Is there a way to set the display name and description using only the package.json?

I don't really care where these are set. It would be nice to have as little redundancy as possible, have a package.json npm is happy with, and had a proper skill name on the web page.